src-v2/NTS.Domain.Core/Configuration/StaticOptions.cs

```diff
@@ -10,7 +10,8 @@ public class StaticOptions : IStartupInitializer, ISingletonService
     
     static Model? _options;
     public static IRegionalConfiguration? RegionalConfiguration { get; set; }
-    public static DetectionConfiguration? DetectionConfiguration { get; set; }
+    public static bool RfidDetectionEnabled => _options != default && _options.DetectionMode == DetectionMode.Rfid;
+    public static bool VisionDetectionEnabled => _options != default &&  _options.DetectionMode == DetectionMode.ComputerVision;
```

Comment:
Lets keep the API for StaticOptions consistent and use methods instead of dynamic properties.